src/gpu/intel/ref_matmul.cl

```diff
@@ -37,17 +37,45 @@ uint get_dropout_threshold(float p) {
 
 __kernel void ref_matmul(__global SRC_DATA_T *A, __global WEI_DATA_T *B,
         __global DST_DATA_T *C, __global BIA_DATA_T *bia,
-        __global SRC_ZP_DATA_T *a0, long src_zp_stride_k, long src_zp_stride_m,
-        long src_zp_group_k, __global WEI_ZP_DATA_T *b0, long wei_zp_stride_n,
-        long wei_zp_stride_k, long wei_zp_stride_d0, long wei_zp_stride_d1,
-        long wei_zp_group_n, long wei_zp_group_k, __global int *c0,
-        __global SRC_SCALES_DATA_T *src_scales, long src_scale_stride_k,
-        long src_scale_stride_m, long src_scale_stride_d0,
-        long src_scale_stride_d1, long src_scale_group_k,
-        __global WEI_SCALES_DATA_T *wei_scales, long wei_scale_stride_n,
-        long wei_scale_stride_k, long wei_scale_stride_d0,
-        long wei_scale_stride_d1, long wei_scale_group_n,
-        long wei_scale_group_k, __global DST_SCALES_DATA_T *dst_scales,
+#if HOST_SRC_ZP
+        int src_zp_value,
```

Comment:
Nit: for consistency `src_zp_value` -> `a0`. Also should be `SRC_ZP_DATA_T` type?
```suggestion
        SRC_ZP_DATA_T a0_value,
```

Reply:
My bad, fixed and renamed